FLEDGE.md

```diff
@@ -803,3 +811,184 @@ These signals were requested in [issue 435](https://github.com/WICG/turtledove/i
 We also need to provide a mechanism for the _losing_ bidders in the auction to learn aggregate outcomes.  Certainly they should be able to count the number of times they bid, and losing ads should also be able to learn (in aggregate) some seller-provided information about e.g. the auction clearing price.  Likewise, a reporting mechanism should be available to buyers who attempted to bid with a creative that had not yet reached the k-anonymity threshold.
 
 This could be handled by a `reportLoss()` function running in the worklet.  Alternatively, the model of [SPURFOWL](https://github.com/AdRoll/privacy/blob/main/SPURFOWL.md) (an append-only datastore and later aggregate log processing) could be a good fit for this use case.  The details here are yet to be determined.
+
+### 6. Additional Bids
+
+In addition to bids generated by interest groups, sellers can enable buyers to introduce bids generated outside of the auction. We refer to these as additional bids. Additional bids are commonly triggered using contextual signals. By having more contextual bids participate in the auction, we're reducing the leakage that's learned by detecting whether or not an auction produced an interest-group-based winning ad.
+
+Buyers compute the additional bids, likely as part of a contextual auction. Buyers need to package up each additional bid using a new data structure that encapsulates all of the information needed for the additional bid to compete against other bids in a Protected Audience auction. Sellers are responsible for passing additional bids to the browser at Protected Audience auction time
+
+Each additional bid is expressed using the following JSON data structure:
+
+```
+const additionalBid = {
+  "bid": {
+    // Fields analogous to those returned by generateBid()
+    "ad": 'ad-metadata',
+    "adCost": 2.99,
+    "bid": 1.99,
+    "bidCurrency": "USD",
+    "render": "https://www.example-dsp.com/ad/123.jpg",
+    "adComponents": [adComponent1, adComponent2],
+    "allowComponentAuction": true,
+    "modelingSignals": 123,
```

Comment:
Since additional bids are not targeted using IG data, and are not subject to k-anon, will the modelingSignals be noised & bucketed?

Reply:
Yes. For consistency, the values passed into `reportAdditionalBidWin()` for additional bids will be the same as those passed into `reportWin()` for winning bids generated from stored interest groups.